package.json

```diff
@@ -5,23 +5,22 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
-		"build": "ENVIRONMENT=production webpack --mode production && yarn install --modules-folder ./build/node_modules --production --frozen-lockfile",
-		"build:analyze": "WEBPACK_ANALYZE_BUNDLE=true yarn build",
+		"build": "ENVIRONMENT=production webpack --mode production && pnpm install --prod --node-linker=hoisted --dir=build --ignore-scripts",
```

Comment:
Are we setting the linker to `hoisted` just because it's safer for production?

Reply:
Yeah it's just the way our production build works. For some reason not using hoisted caused it to fail. Need to investigate whether this is something we can change, but for now this replicates the existing behaviour.